test/DynamoCoreWpfTests/SliderViewModelTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Windows.Controls;
+using CoreNodeModels;
+using CoreNodeModels.Input;
+using CoreNodeModelsWpf;
+using CoreNodeModelsWpf.Controls;
+using Dynamo.Nodes;
+using Dynamo.Tests;
+using Dynamo.Utilities;
+using DynamoCoreWpfTests.Utility;
+using NUnit.Framework;
+
+namespace DynamoCoreWpfTests
+{
+    [TestFixture]
+    class SliderViewModelTests : DynamoTestUIBase
+    {
+        private AssemblyHelper assemblyHelper;
+
+        public override void Open(string path)
+        {
+            base.Open(path);
+
+            DispatcherUtil.DoEvents();
+        }
+
+        public override void Run()
+        {
+            base.Run();
+
+            DispatcherUtil.DoEvents();
+        }
+
+        protected override void GetLibrariesToPreload(List<string> libraries)
+        {
+            libraries.Add("VMDataBridge.dll");
+            libraries.Add("DSCoreNodes.dll");
+            base.GetLibrariesToPreload(libraries);
+        }
+
+        /// <summary>
+        /// This test method will validate the next properties from the SliderViewModel class:
+        ///  public T Max
+        ///  public T Min
+        ///  public T Step
+        /// </summary>
+        [Test]
+        public void SliderViewModel_PropertiesTest()
+        {
+            Open(@"core\library\NumberSliderNodeTest.dyn");
+
+            var nodeView = NodeViewWithGuid("17ef3a6e-4742-45b6-8a2e-7ec3901e7726");//NodeViewOf<DoubleSlider>();
```

Comment:
Please remove the unused code

Reply:
@QilongTang 
It was a comment, then I re-wrote the comment so it doesn't look like unused code.
See the commit: https://github.com/DynamoDS/Dynamo/pull/11217/commits/77da3f7f9899682b209e1221f198809a2f2be318